app/assets/javascripts/viewer.js

```diff
@@ -0,0 +1,68 @@
+function loadUV () {
+  let urlDataProvider = new UV.URLDataProvider(false)
```

Comment:
It looks like the scoping here is not right. See the deploy failure: https://gist.github.com/tpendragon/bcaa3d58547a04e6c78a28b9b907650a

Reply:
@hackmastera It was just the `let` vs a `var`. Should be fixed now - just tested a deploy.